### PR TITLE
feat(content-block): added mock markup for content-block-heading

### PR DESCRIPTION
**Pull request by RaphaelAmadeu**
### Related Ticket(s)

#6083 

### Description

 I've added some light dom rendering for content-block-heading and content-block-paragraph, so its content can be accessed and read by SEO tools.

I've also made a utility that strips HTML from a string, as we needed that to get the content to be rendered.

### CODE REVIEW

File: packages/web-components/src/components/content-block/content-block-heading.ts [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+ * Copyright IBM Corp. 2020, 2021
  *
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
 
-import { html, property, customElement, LitElement } from 'lit-element';
+import { html, property, customElement, LitElement, internalProperty } from 'lit-element';
 import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import { stripHTML } from '@carbon/ibmdotcom-utilities/es/utilities/stripHTML/index.js';
```
Comment (jeffchew) [inline packages/web-components/src/components/content-block/content-block-heading.ts:12]: @RaphaelAmadeu I don't see the utility, forgot to add it?

Comment (RaphaelAmadeu) [reply packages/web-components/src/components/content-block/content-block-heading.ts:12]: corrected this in the latest commit

File: packages/utilities/src/utilities/stripHTML/stripHTML.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ *
+ * @example
+ * import { stripHTML } from '@carbon/ibmdotcom-utilities';
+ *
+ * content = stripHTML(this.innerHtml);
+ *
+ * @param {string} content with html tags
+ * @returns {null} content without html tags
+ */
+const stripHTML = content => content.replace(/(<([^>]+)>)/gi, '');
```
Comment (jeffchew) [inline packages/utilities/src/utilities/stripHTML/stripHTML.js:19]: @RaphaelAmadeu can you create some light unit tests around this utility?

File: packages/web-components/src/components/content-block/content-block-heading.ts [hunk 3]
```diff
@@ -37,6 +42,16 @@ class DDSContentBlockHeading extends StableSelectorMixin(LitElement) {
     super.connectedCallback();
   }
 
+  firstUpdated() {
+    this.content = stripHTML(this.innerHTML);
+    render(
+      html`
+        <h2>${this.content}</h2>
```
Comment (jeffchew) [inline packages/web-components/src/components/content-block/content-block-heading.ts:49]: @RaphaelAmadeu the additional heading markup is causing the actual render to look a bit off. If we just do the `stripHTML` and not add the `h2`, how does Bing renderer capture the lightDOM?

File: packages/web-components/src/components/content-block/content-block-paragraph.ts [hunk 4]
```diff
@@ -20,9 +22,22 @@ const { stablePrefix: ddsPrefix } = ddsSettings;
...
 @customElement(`${ddsPrefix}-content-block-paragraph`)
 class DDSContentBlockParagraph extends LitElement {
+  @internalProperty()
+  content = '';
+
+  firstUpdated() {
+    this.content = stripHTML(this.innerHTML);
+    render(
+      html`
+        <p>${this.content}</p>
```
Comment (jeffchew) [inline packages/web-components/src/components/content-block/content-block-paragraph.ts:32]: I'm wondering if adding the `<p>` tag is a good idea here, can we just run the `stripHTML` without adding additional wrapper tags? I think the `<p>` tag also wouldn't really bring any benefit from an SEO perspective.

File: packages/utilities/src/utilities/stripHTML/stripHTML.js (hunk 2 above)
Comment (RaphaelAmadeu) [reply packages/utilities/src/utilities/stripHTML/stripHTML.js:19]: @jeffchew i've added this in the latest commit

File: packages/utilities/src/utilities/stripHTML/stripHTML.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+ * @returns {null} content without html tags
+ */
+const stripHTML = content => {
+  const component = document.createElement('textarea');
+  component.innerHTML = content;
+
+  return component.childNodes[0].nodeValue.replace(/(<([^>]+)>)/gi, '');
+};
+
+export default stripHTML;
```
Comment (jeffchew) [inline packages/utilities/src/utilities/stripHTML/stripHTML.js:26]: @RaphaelAmadeu one more thing, can you add a unit test for this utility?

**Review by jeffchew — APPROVED** [review]
Thank you @RaphaelAmadeu !

**Pull request merged**